src/modules/GroupChannel/components/MessageList/InfiniteList.tsx

```diff
@@ -87,7 +87,7 @@ export const InfiniteList = forwardRef((props: Props, listRef: React.RefObject<H
     } else {
       direction.current = undefined;
     }
-  };
+  }, [listRef.current, messages.length]);
```

Comment:
How about using `listRef.current?.scrollHeight, listRef.current?.scrollTop, listRef.current?.clientHeight` instead of `listRef.current`?

Reply:
Oh actually we don't even need to add `ref.current` in here cause it's a mutable object. So I removed it. https://github.com/sendbird/sendbird-uikit-react/pull/1288/commits/f9168a03b7c020a0dd114cac2e3a512a25534b1a#diff-5304e2564a5f57c7d83cf050597f6f595beef659c04a8156d25b1e203bd2281bL90